### ISSUE TITLE
getBid() of root ProductPartition returns NULL

### ISSUE THREAD

**Issue by TerminusMKB** [googleads/googleads-java-lib#36]
If AdGroup containts only one ProductPartition (root), and it has BIDDABLE_UNIT type, ProductPartitionTree generates root element with bid = NULL:

```
ProductPartitionTree partitionTree = ProductPartitionTree.createAdGroupTree(new AdWordsServices(), adWordsSession, request.getAdGroupId());
ProductPartitionNode rootNode = partitionTree.getRoot();
rootNode.getBid(); //returns NULL, but really it has valid value in web interface
```

**Comment by jradcliff** [comment]
Thanks for filing this bug per our forum discussion. I'll take a look at this and try to get a fix in for it in the next release of the library.

**Comment by jradcliff** [comment]
Fixed in [v1.38.0](https://github.com/googleads/googleads-java-lib/releases/tag/1.38.0).
